### PR TITLE
Add "View page source" repo link

### LINKED ISSUE #756: Add "View page source" repository link
Context: [Repository Links](https://www.docsy.dev/docs/adding-content/repository-links/)

A common workflow that I've encountered is for folks to view a page's source before they decide whether they'd like to submit an edit.

A problem with **Edit this page** is that is forces the user to login _and_ have a fork of the repo before submitting an edit (admittedly, GH will fork the repo for you). IMHO, user's should be forced to login before they can view a page.

### PR DESCRIPTION
**Pull request by chalin**
- Closes #756
- Contributes to #62 
- Contributes to #731 by adding unique classes to each link so that they can be individually excluded
- Added warning that `github_url` is deprecated (cf. #737)

### CODE REVIEW

**Comment by chalin** [comment]
@LisaFC @emckean - if you agree with these changes, then I'll also submit an update to the docs, but I wanted your thumbs up first.

File: layouts/partials/page-meta-links.html [hunk 1]
```diff
@@ -29,19 +30,20 @@
     {{ $gh_repo_path = replaceRE . $ghs_rename $gh_repo_path -}}
   {{ end -}}
 
+  {{ $viewURL := printf "%s/tree/%s" $gh_repo $gh_repo_path -}}
   {{ $editURL := printf "%s/edit/%s" $gh_repo $gh_repo_path -}}
-  {{ $createURL := printf "%s/edit/%s" $gh_repo $gh_repo_path -}}
```
Comment (chalin) [inline layouts/partials/page-meta-links.html:33]: I deleted this variable assignment because `$createURL` doesn't appear to be used anymore.

**Comment by LisaFC** [comment]
@emckean thoughts? Looks good I think.

**Comment by chalin** [comment]
@emckean @LisaFC - friendly ping

**Comment by LisaFC** [comment]
Looks great! Merging.

**Pull request merged**